deployment/postgres-init-db/sql/sched/tables/scheduling_rule.sql

```diff
@@ -0,0 +1,57 @@
+create table scheduling_rule (
+  id integer generated always as identity,
+  revision integer not null default 0,
+  definition jsonb not null,
+
+  model_id integer not null,
+  description text null,
+  author text null,
+  last_modified_by text null,
+  created_date timestamptz not null default now(),
+  modified_date timestamptz not null default now(),
+
+  constraint scheduling_rule_synthetic_key
+    primary key (id)
+-- TODO `references mission_model` disabled for now, must refer to `aerie_merlin` DB
+--  constraint scheduling_rule_references_mission_model
```

Comment:
This was lifted from @kroffo's initial tables for the scheduler. I've commented out this section since it refers to the `mission_model` table. It's expected that Kenny will uncomment and fix this as part of his work.

Reply:
I think if Kenny is to add/implement this it should first appear with along with the code that implements it. So we can remove it entirely from this PR.

Separately, given the scheduler db and the merlin db are separated by an Aerie module boundary (different components) we won't be able to refer to an merlin db tables. Is this intended to be literal or more of a reminder that we will need to implement a mechanism to manage the mission model id to which a scheduling rule will be associated?

Reply:
Sounds good, I'll remove this :) And it was meant to be a reminder that the `aerie_sched` DB and `aerie_merlin` DB are not to be coupled so that `constraint` would need to be rewritten, removed, or reworked somehow.

Reply:
Wonderful. And thanks for sharing the context!